scripts/tgtg_auth.py

```
@@ -0,0 +1,87 @@
+import threading
+import time
+from pathlib import Path
+
+# Add project root to path
+sys.path.insert(0, str(Path(__file__).parent.parent))
+
+from tgtg import TgtgClient
+from src.universal_agent.tgtg.config import CREDENTIALS_FILE
+
+EMAIL = "kevinjdragan@gmail.com"
+STATE_FILE = Path("/tmp/tgtg_auth_state.json")
```

Ruff (0.15.2)

[error] Probable insecure usage of temporary file or directory: "/tmp/tgtg_auth_state.json"

(S108)